lib/src/pages/card_page.dart

```diff
@@ -0,0 +1,163 @@
+              decoration: BoxDecoration(
+                gradient: RadialGradient(
+                  radius: Constants.gradientRadius,
+                  colors: [
+                    Colors.purple,
+                    Colors.deepPurple,
+                  ],
+                ),
+                color: Colors.deepPurpleAccent,
+                borderRadius: BorderRadius.circular(
+                  Constants.borderRadius,
+                ),
+                border: Border.all(
+                  color: Colors.deepPurple,
+                  width: Constants.borderWidth,
+                ),
+              ),
+              child: Column(
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.only(
+                      top: Constants.circleAvatarPaddingTop,
+                    ),
+                    child: CircleAvatar(
+                      radius: Constants.circleAvatarRadius,
+                      backgroundImage: AssetImage(
+                        this.srcImage,
+                      ),
+                    ),
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.only(
+                      top: Constants.containerPaddingTop,
+                    ),
+                    child: Container(
+                      child: Text(
+                        this.name,
+                        style: TextStyle(
+                          fontSize: Constants.fontSize,
+                          color: Constants.cardTextColor,
+                        ),
+                      ),
+                    ),
+                  ),
+                  Container(
+                    padding: EdgeInsets.only(
+                      top: Constants.paddingPosition,
+                    ),
+                    child: Text(
+                      this.position,
+                      style: TextStyle(
+                        color: Constants.cardTextColor,
+                        fontSize: Constants.fontSize,
+                        fontStyle: FontStyle.italic,
+                      ),
+                    ),
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.only(
+                      top: Constants.containerPaddingTop,
+                    ),
+                    child: Container(
+                      child: Divider(
+                        color: Colors.deepPurpleAccent,
+                        thickness: Constants.dividerThickness,
+                      ),
+                    ),
+                  ),
+                  Card(
+                    elevation: Constants.cardElevation,
+                    color: Colors.purple,
+                    child: ListTile(
+                      leading: Icon(
+                        Icons.phone_android,
+                        color: Constants.cardTextColor,
+                      ),
+                      title: Text(
+                        this.number,
```

Comment:
Inside CardPage is not neccesary to use "this" to call the defined variables

Reply:
Fixed!